scripts/docs/generate-transformations.ts

```diff
@@ -45,6 +45,7 @@ labels:
     - enterprise
     - oss
 title: Transform data
+description: Use transformations to manipulate your data before visualizations are applied
```

Comment:
@imatwawana this template builds the content in `docs/sources/panels-visualizations/query-transform-data/transform-data/index.md`, so I'm also updating here. :)

Reply:
And if you update with @nmarrs suggestions, please also update this.

Reply:
I wasn't sure about whether I needed to use this file to update frontmatter. Now I know, lol

Reply:
No worries! :)